### PR TITLE
docs: Quickstart cleanup

### PR DESCRIPTION
**Pull request by rube-de**
This PR
- removes truffle
- makes initial setup clearer
- changes diff to normal clodeblock with highliting for better copy

our prism dark theme, which is applied on the code-blocks, isn't the best visible. Maybe we could change from dracula to vsDark

### CODE REVIEW

File: docs/quickstart.mdx [hunk 1]
```diff
@@ -48,43 +34,38 @@ Let's make it happen!
 We're going to use Hardhat, but Sapphire should be compatible with your dev
 environment of choice. Let us know if things are not as expected!
 
-1. Make & enter a new directory
-2. `npx hardhat@~2.19.2 init` then create a TypeScript project.
+1. Make & enter a new directory `mkdir quickstart && cd quickstart`
+2. `npx hardhat@~2.19.2 init` then create a TypeScript project and install the project dependencies.
```
Comment (aefhm) [inline docs/quickstart.mdx:38]: Could we sneak in this bump? https://www.npmjs.com/package/@oasisprotocol/sapphire-hardhat?activeTab=versions

```suggestion
2. `npx hardhat@~2.19.4 init` then create a TypeScript project and install the project dependencies.
```

**Comment by aefhm** [comment]
> our prism dark theme, which is applied on the code-blocks, isn't the best visible. Maybe we could change from dracula to vsDark

Agree.

**Review by aefhm — APPROVED** [review]

File: docs/quickstart.mdx [hunk 2]
```diff
@@ -48,43 +34,38 @@ Let's make it happen!
...
-+    'sapphire-testnet': {
-+      // This is Testnet! If you want Mainnet, add a new network config item.
-+      url: "https://testnet.sapphire.oasis.io",
-+      accounts: process.env.PRIVATE_KEY
-+        ? [process.env.PRIVATE_KEY]
-+        : [],
-+      chainId: 0x5aff,
-+    },
-+  },
+  solidity: "0.8.19",
```
Comment (matevz) [inline docs/quickstart.mdx:56]: This should work fine.

```suggestion
  solidity: "0.8.24",
```

Related: https://github.com/rust-ethereum/evm/issues/277, https://github.com/oasisprotocol/oasis-sdk/issues/1853

**Review by matevz — APPROVED** [review]

**Comment by matevz** [comment]
Don't forget to squash before merging.

Also, please rebase the PR on the latest main branch instead of merging main changes into your PR.

**Comment by rube-de** [comment]
> > our prism dark theme, which is applied on the code-blocks, isn't the best visible. Maybe we could change from dracula to vsDark
> 
> Agree.

alternative, we could set
[data-theme='dark'] span.token-line.theme-code-block-highlighted-line span.token.comment { 
color: /*somthing better visible like greenish*/
}
to change the color of the highlighted comment

**Comment by aefhm** [comment]
> alternative, we could set
> [data-theme='dark'] span.token-line.theme-code-block-highlighted-line span.token.comment {
> color: /_somthing better visible like greenish_/
> }
> to change the color of the highlighted comment

I'm open to any of these. I think the highlighting you introduced already is a lot better.

**Pull request merged**